### PR TITLE
Add event for claim in rater

### PR DESCRIPTION
**Pull request by MiksuJak**

### CODE REVIEW

File: contracts/truefi/TrueRatingAgency.sol [hunk 1]
```diff
@@ -85,6 +85,7 @@ contract TrueRatingAgency is ITrueRatingAgency, Ownable {
     event LoanRetracted(address id);
     event Voted(address loanToken, address voter, bool choice, uint256 stake);
     event Withdrawn(address loanToken, address voter, uint256 stake, uint256 received, uint256 burned);
+    event Claimed(address loanToken, address voter, uint256 claimableReward);
```
Comment (barrutko) [inline contracts/truefi/TrueRatingAgency.sol:88]:
```suggestion
    event Claimed(address loanToken, address voter, uint256 claimedReward);
```

**Review by barrutko — APPROVED** [review]
One minor fix to be applied and besides that looks fine 👌

**Comment by MiksuJak** [comment]
Needs additional review because testing method has changed

File: test/truefi/TrueRatingAgency.test.ts [hunk 2]
```diff
@@ -719,6 +720,20 @@ describe('TrueRatingAgency', () => {
...
+      await loanToken.fund()
+      await timeTravel(monthInSeconds * 6)
+
+      const tx = await rater.claim(loanToken.address, owner.address, txArgs)
+      const receipt = await tx.wait()
+      const event = new utils.Interface(TrueRatingAgencyJson.abi).parseLog(receipt.events[2])
+
+      expect(event.args[0]).eq(loanToken.address)
+      expect(event.args[1]).eq(owner.address)
+      expectCloseTo(BigNumber.from(event.args[2]), BigNumber.from(25000000000000))
```
Comment (barrutko) [inline test/truefi/TrueRatingAgency.test.ts:734]:
```suggestion
      expectCloseTo(BigNumber.from(event.args[2]), parseTT(250000))
```

**Review by barrutko — APPROVED** [review]
One last thing I'd change.

**Pull request merged**